policy-controller/k8s/status/src/index.rs

```diff
@@ -191,7 +202,9 @@ impl Index {
                         port: None,
                     },
                     controller_name: POLICY_CONTROLLER_NAME.to_string(),
-                    conditions: vec![condition],
+                    // Servers may not have backend references so they
```

Comment:
after thinking about this some more, I think we probably want to just omit any kind of backend status condition for parents which are not Services.  Another way to think about this is that we only try to resolve the backend_refs when the parent is a Service.  For all other parents we just ignore the backend_refs so I don't think we need to report any status conditions there.